### ISSUE TITLE
Please rename this VTKHDF implementation

### ISSUE THREAD

**Issue by mwestphal** [jmag722/vtk-hdf#31]
Hi @jmag722 ! I'm a VTK maintainer avec very active in regard to the VTKHDF format.

Thanks a lot for creating this VTKHDF implementation, Its great to see the community embracing it.

However, can I suggest a rename of this project?

VTKHDF is the name of the format.
vtkhdf is the name of the implementation available in the vtkIOHDF module in VTK.

I'm afraid adding yet another similar name will be confusing, wdyt ?

**Comment by jmag722** [comment]
Hello @mwestphal ,

It's really an amazing format. I'm open to renaming it, however with VTKHDF being a block of consonants, it was hard to think of a name that was unique but pithy

**Comment by mwestphal** [comment]
Unless I'm mistaken your project seems to be aiming at using python to create VTKHDF files. How about pyvtkhdf ? vtkhdf-py ?

**Comment by jmag722** [comment]
yeah, those are definitely clear. trying to think of shorter names that are still meaningful.

pyv5
v5py
pyvis5
pv5
v5k
v5kpy
pyvi5

then there's the risk of, what if I want to write a VTK HDF interface in another language. it'd be nice to be able to consolidate in 1 package without changing repo name
